scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -54,15 +53,12 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     };
     this.activeFilterTags = {};
 
-    if (appConfig.features.hideCreatePackage) {
-        availableFileTypes.forEach((file) => {
-            if (file.type !== 'composite' && file.type !== 'highlight-pack') {
-                this.fileTypes.push(file);
-            }
-        });
-    } else {
-        this.fileTypes = availableFileTypes;
-    }
+    this.fileTypes = this.fileTypes.filter(
+        (fileType) => (
+            appConfig.features.hideCreatePackage ?
+                fileType.type !== 'composite' && fileType.type !== 'highlight-pack' :
+                fileType
```

Comment:
it would be more readable if you put here `true`, if the config is not set there is no filtering

Reply:
You mean something like this: 
`appConfig.features.hideCreatePackage ?
                fileType.type !== 'composite' && fileType.type !== 'highlight-pack' :
                true`
 or something like this:
`appConfig.features.hideCreatePackage === true ?
                fileType.type !== 'composite' && fileType.type !== 'highlight-pack' :
                fileType`

Reply:
I mean the first one